docs/faq.md

```diff
@@ -15,6 +19,37 @@ using standard HTTP compression negotiation and eliminating binary framing from
 the body, the Connect protocol lets us make unary RPCs with web browsers, cURL,
 or any other HTTP client.
 
+### How do I use the Connect protocol to call existing gRPC servers?
+
+Most Connect implementations support the gRPC protocol, so you have a choice:
+
+- Use the Connect runtime, but configure your client to use the gRPC protocol, or
+- Use the Connect runtime and Connect protocol, and have Envoy automatically
+  handle the Connect-to-gRPC translation. Envoy is a popular and widely used
+  proxy.
+
+Envoy v1.26 ships with a Connect-gRPC bridge that allows clients to speak the
+Connect protocol (including GET requests) to existing gRPC servers. You can find
+a demo here: https://github.com/connectrpc/envoy-demo
+
+### How do I reliably call a server streaming RPC from a web browser?
+
+The answer is highly dependent on all of the networking parties involved. Generally,
+make sure that your server or your infrastructure does not apply timeouts within
+the expected duration of calls. If possible, pre-empt timeouts by setting short
+deadlines and by repeating the call when the deadline is exceeded. Read the
+[streaming docs](go/streaming.md) for the Go implementation to get an idea of
+the implications.
+
+### How do I proxy the Connect protocol through NGINX?
+
+Request/response (unary) RPCS can be proxied through NGINX using Connect
+as long as they don't require end-to-end HTTP/2, which NGINX doesn't support.
+Bidirectional and client streaming can't be proxied through NGINX either
+because they require HTTP/2, so we recommend using Envoy Proxy, Apache, or
+TCP-level load balancers like HAProxy, which support all of the Connect
+protocol's features.
```

Comment:
Suggest this:

"Request-response (unary) RPCs made with the Connect protocol can be proxied through NGINX, because they don't require end-to-end HTTP/2. Streaming RPCs typically require end-to-end HTTP/2, which NGINX doesn't support. Rather than NGINX, we recommend using Envoy, Apache, or TCP-level load balancers like HAProxy, all of which support the full Connect protocol."